Tasks/ECRPullImage/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "ecrpullimage",
+    "version": "1.1.8",
+    "description": "Pulls a Docker image from Elastic Container Registry",
+    "main": "ECRPullImage.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
What does this do?

Reply:
Good question, this is one of the things I haven't gotten around to deleting (there is one per task), but I think the answer is nothing. I wasn't entirely sure about that when I started working on VSTS, but I think they can all be removed.

Reply:
Mystery solved, it is used by webpack to package properly, so it is needed.